pandas/tests/window/test_rolling.py

```diff
@@ -130,29 +130,94 @@ def test_closed_fixed(closed, arithmetic_win_operators):
     df_fixed = DataFrame({"A": [0, 1, 2, 3, 4]})
     df_time = DataFrame({"A": [0, 1, 2, 3, 4]}, index=date_range("2020", periods=5))
 
-    result = getattr(df_fixed.rolling(2, closed=closed, min_periods=1), func_name)()
-    expected = getattr(df_time.rolling("2D", closed=closed), func_name)().reset_index(
-        drop=True
-    )
+    result = getattr(
+        df_fixed.rolling(2, closed=closed, min_periods=1, center=False), func_name
+    )()
+    expected = getattr(
+        df_time.rolling("2D", closed=closed, min_periods=1, center=False), func_name
+    )().reset_index(drop=True)
 
     tm.assert_frame_equal(result, expected)
 
 
-def test_closed_fixed_binary_col():
+def test_datetimelike_centered_selections(closed, arithmetic_win_operators):
```

Comment:
can you paramterize on closed (e.g. directly put your window selection up there with the closed parameter),  eg.. don't use the fixture

Reply:
Good idea, this is done, too.